sdks/java/core/src/test/java/org/apache/beam/sdk/runners/inprocess/EncodabilityEnforcementFactoryTest.java

```diff
@@ -56,7 +56,7 @@
   public void encodeFailsThrows() {
     TestPipeline p = TestPipeline.create();
     PCollection<Record> unencodable =
-        p.apply(Create.of(new Record()).withCoder(new RecordNoEncodeCoder()));
+        p.apply(Create.<Record>of().withCoder(new RecordNoEncodeCoder()));
```

Comment:
Maybe just refactor the common code to a utility method to make it clear what is being tested in this and below tests?

The key IIUC is that this is just a unit test of ModelEnforcement, and all of the setup code is just to get the ability to create bundles of the test record in the right way.